### DISCUSSION TITLE
Showing a list of recently created/updated pages

### DISCUSSION THREAD

**Discussion by k8jisoo** [requarks/wiki#6739] [Help / Questions] [answered]
Is there a way to show a list of recently created/updated pages on a particular page?

**Comment by NGPixel** [comment, marked as answer]
Not at the moment.

**Comment by FarmDoc** [comment]
I am not sure if I understand you question, but if you want to see newly created or newly edited pages, you can do this in the Admin view.
(just sort the "Pages" list accordingly)

Or do you want this list in the wiki itself?
